### PR TITLE
Use bitshift consistently for bitflag definitions

### PR DESCRIPTION
**Pull request by sbrugman**
## Summary

Most places in the code use bit shifts to set bitflag values, apart from these few.
Equivalent code, just more consistent and easier to add new flags.

Found these when defining new bitflags in another PR.

### CODE REVIEW

File: crates/ruff/src/printer.rs [hunk 1]
```diff
@@ -27,11 +27,11 @@ bitflags! {
     #[derive(Default, Debug, Copy, Clone)]
     pub(crate) struct Flags: u8 {
         /// Whether to show violations when emitting diagnostics.
-        const SHOW_VIOLATIONS = 0b0000_0001;
+        const SHOW_VIOLATIONS = 1 << 0;
         /// Whether to show a summary of the fixed violations when emitting diagnostics.
-        const SHOW_FIX_SUMMARY = 0b0000_0100;
+        const SHOW_FIX_SUMMARY = 1 << 2;
```
Comment (sbrugman) [inline crates/ruff/src/printer.rs:32]: Is there any particular reason to not change this to `1 << 1`?

Comment (MichaReiser) [reply crates/ruff/src/printer.rs:32]: It should be fine. I suspect that we used to have more flags and didn't update the values when we removed one.

File: crates/red_knot_python_semantic/src/semantic_index/symbol.rs
Comment (MichaReiser) [inline crates/red_knot_python_semantic/src/semantic_index/symbol.rs:1]: I would prefer if we can keep the formatting as is to reduce unnecessary changes. The same applies for crates/ruff_linter/src/message/text.rs

**Review by MichaReiser — APPROVED** [review]
Thanks
